tester.go

```diff
@@ -20,6 +21,11 @@ type Tester struct {
 func newTester(env map[string]string, definition tester_definition.TesterDefinition) (Tester, error) {
 	context, err := tester_context.GetTesterContext(env, definition)
 	if err != nil {
+		if friendlyErr, ok := err.(*internal.FriendlyError); ok {
+			fmt.Printf(friendlyErr.UserError)
```

Comment:
Do we actually need to print this twice? Doesn't the returned error (below) get printed anyway?